test_data/saml_config.yml

```diff
@@ -0,0 +1,14 @@
+---
+IDP_METADATA_PATH: |
+    <xml> this is some idp xml data </xml>
+DISPLAY_NAME_ATTRIBUTE: 'def'
+GROUP_ATTRIBUTE: 'ghi'
+MAX_AUTH_LIFETIME: 10
+USERNAME_ATTRIBUTE: 'jkl'
+EMAIL_ATTRIBUTE: 'mno'
+LOGOUT_URL: 'https://www.example.com/logout'
+ADVANCED_CONFIGURATION: []
+ENCRYPTION_DATA: []
```

Comment:
From the code above, it looks like these are mappings; should they use `{}` instead here?